src/AppInstallerCLICore/Workflows/ShowFlow.h

```diff
@@ -11,6 +11,18 @@ namespace AppInstaller::CLI::Workflow
     // Outputs: None
     void ShowManifestInfo(Execution::Context& context);
 
+    // Shows information on an application that is global to all installers.
+    // Required Args: None
+    // Inputs: Manifest
+    // Outputs: None
+    void ShowManifestGlobalInfo(Execution::Context& context);

```

Comment:
>ShowManifestGlobalInfo [](http://example.com/codeflow?start=9&length=22)

nit: to be more consistent in naming, this could be ShowPackageInfo (corresponding to below ShowInstallerInfo)